backend/src/main/java/com/crm/model/db/CarEntity.java

```diff
@@ -1,7 +1,6 @@
 package com.crm.model.db;
 
-import lombok.Getter;
-import lombok.NoArgsConstructor;
+import lombok.*;
```

Comment:
separate import

Reply:
I don't agree with that as it changes automatically everytime you add a new import from lombok library.

Reply:
W intellij możesz zmienić to ustawienie

Reply:
Ok dzięki, poprawiłem.